compiler/grainlsp/utils.re

```diff
@@ -0,0 +1,816 @@
+open Grain;
+open Compile;
+open Grain_parsing;
+open Grain_utils;
+open Grain_typed;
+open Grain_diagnostics;
+
+//  CompletionItemKind
+let item_kind_completion_text = 1;
+let item_kind_completion_function = 3;
+let item_kind_completion_constructor = 4;
+let item_kind_completion_variable = 6;
+let item_kind_completion_struct = 22;
+
+let markdown_grain = code => "```grain\n" ++ code ++ "\n```";
```

Comment:
Don't I have this as a utility in `Grain_utils.Markdown` already?

Reply:
Yes you do, all references switched to that